src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/LocationGroupServiceSelectorTest.kt

```diff
@@ -5,12 +5,10 @@ import com.nhaarman.mockito_kotlin.verify
 import com.nhaarman.mockito_kotlin.verifyNoMoreInteractions
 import com.nhaarman.mockito_kotlin.whenever
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.mockito.junit.MockitoJUnitRunner.StrictStubs
+import org.junit.jupiter.api.Test
 import uk.gov.justice.digital.hmpps.whereabouts.model.LocationGroup
 
-@RunWith(StrictStubs::class)
+//@RunWith(StrictStubs::class)
```

Comment:
@ExtendWith(MockitoExtension.class)
@MockitoSettings(strictness = Strictness.STRICT_STUBS) ?

Reply:
Ah good spot, forgot to remove the commented out code.  I think all stubs should be strict so will look at adding this to all tests in the future. Or even better look for a way to make them all strict by default.